_posts/2022-08-29-hello-evvnt.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Hello Evvnt"
+date:   2022-08-29 15:50:14 -0400
+categories: engineering
+---
+Welcome to the Evvnt Engineering Blog! This is our hello world introducing Evvnt and what we do. First thing is first, its not a typo! The company name is Evvnt and its pronounced Event. (We don't blame you if you pronounce it like [marwage](https://www.youtube.com/watch?v=3odMTPuzLwY).)
```

Comment:
Welcome to the Evvnt Engineering Blog! This is our hello world introducing Evvnt and what we do. First things first, it's not a typo! The company name is Evvnt and it's pronounced Event. (We don't blame you if you pronounce it like [marwage](https://www.youtube.com/watch?v=3odMTPuzLwY).)